HHS/Head-Start-TTADP
# [Prod] Rename RTTAPA tab, add permission to user export, accessibility fixes

## Description of change
- Rename "Goals & Objectives" tab on RTR to "RTTAPA"
- Add the new training report permissions to the "export all users" CSV download
- Fix the HTML of the topics guidance drawer to improve keyboard navigation and screenreader experience.

## How to test

- Confirm that the tab is renamed
- Check the new permissions are on the CSV export
- Use a screenreader to open the topics guidance. Alternately, if you click on the "Topics *" label text, it will now focus on the dropdown, as expected, instead of the button to toggle the guidance drawer.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1887
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1850
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1922


## Checklists

### Every PR



### Production Deploy


### After merge/deploy
